src/Band.c

```diff
@@ -52,7 +52,7 @@ DmDlsInstrument* DmInstrument_getDlsInstrument(DmInstrument* slf) {
 	uint32_t patch = slf->patch & 0xFFU;
 
 	DmDlsInstrument* ins = NULL;
-	for (long long i = slf->dls->instrument_count; i >= 0; --i) {
```

Comment:
Some DLS files in NOLF only contain one instrument, so I found this fairly quickly.